internal/pkg/crawl/worker.go

```diff
@@ -78,7 +77,7 @@ func (w *Worker) Run() {
 				w.state.lastAction = "waiting for queue to be filled"
 			}
 			if (w.pool.Crawl.Paused.Get() || w.pool.Crawl.Queue.Empty.Get()) && w.pool.Crawl.Queue.CanDequeue() {
```

Comment:
Most network requests cannot be completed within 10ms.

Based on this, I think the 10ms polling will not affect the performance of the worker. And if the queue is not empty, `time.Sleep()` will not be executed.

Reply:
I don't understand the reasoning behind the network request, please elaborate

Reply:
Since we are polling every 10ms here, it will take 10ms for the woker to react to the change of `(w.pool.Crawl.Paused.Get() || w.pool.Crawl.Queue.Empty.Get()) && w.pool.Crawl.Queue.CanDequeue()` from false to true.

I mean 10ms is reasonable.